onnx/backend/test/report/__init__.py

```diff
@@ -6,6 +6,7 @@
 from __future__ import unicode_literals
 
 import pytest  # type: ignore
+import _pytest  # type: ignore
```

Comment:
FYI: I made a commit to ignore _pytest because mypy doesn't have a stub for it. Sorry about directly modifying it because I think your fix is good to be included in the upcoming 1.11 release (so I want to forward this PR ASAP). Thank you for the contribution!